src/app/Main/Body/QueryPage/QueryPageContent-Layout.component.tsx

```diff
@@ -15,22 +15,15 @@ import { QueryPageContentBarChartResultsComponent } from "./Results/Charts/Query
 const contentStyle = css`
     width: 0;
     height: inherit;
-    margin-right: 1em;
-    margin-left: 1em;
     padding: 2em;
     flex: 1 1 0;
     hyphens: auto;
     box-sizing: border-box;
     overflow: auto;
 `;
 const contentBoxStyle = css`
-    max-width: 60em;
-    min-width: 20em;
+    max-width: 45em;
     margin: auto;
-    display: flex;

```

Comment:
Good, block elements are already layouted in a column, so using a flex is only necessary if you want to use wrapping or distribute the space between the block elements in a specific way.